plugins/PermissionsEx/permissions.yml

```diff
@@ -156,6 +160,7 @@ groups:
     - essentials.togglejail
     - essentials.top
     - essentials.tp
+    - essentials.tpo
```

Comment:
@MiniDigger helpers don't have tpo, mods do.
(Mods are usually staff from spigotmc so I assume they can be trusted)